persistence-modules/java-mongodb/src/test/java/com/baeldung/ordering/caseinsensitive/CaseInsensitiveOrderingLiveTest.java

```diff
@@ -0,0 +1,97 @@
+package com.baeldung.ordering.caseinsensitive;
+
+import com.mongodb.MongoClient;
+import com.mongodb.client.*;
+import com.mongodb.client.model.Collation;
+import com.mongodb.client.model.Projections;
+import com.mongodb.client.model.Sorts;
+import org.bson.Document;
+import org.bson.conversions.Bson;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.MongoDBContainer;
+import org.testcontainers.utility.DockerImageName;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static com.mongodb.client.model.Aggregates.project;
+import static com.mongodb.client.model.Aggregates.sort;
+import static com.mongodb.client.model.Sorts.ascending;
+import static org.junit.Assert.assertEquals;
+
+public class CaseInsensitiveOrderingLiveTest {
+
+    private static MongoCollection<Document> userCollections;
+
+    private static MongoDBContainer mongoDBContainer;
+
+    @BeforeAll
+    public static void setup() {
+        mongoDBContainer = new MongoDBContainer(DockerImageName.parse("mongo:4.0.10"));
```

Comment:
You don't need to do this when you use the plugin - you just `new` the container in the initializer list of the class